src/api/templates/searchTemplate.js

```diff
@@ -51,64 +51,59 @@ export function filterEngagementType(engagementTypes) {
     return filters;
 }
 
-export function filterLocation(counties, municipals) {
-    const filters = {
-        bool: {
-            should: []
-        }
-    };
-    if (counties && counties.length > 0) {
-        counties.forEach((county) => {
-            filters.bool.should.push({
-                term: {
-                    county_facet: county
+/**
+ * Lager filter for fasetter med flere nivå, feks fylke og kommune. Kobinerer AND og OR.
```

Comment:
Skriveleif: Kobinerer -> Kombinerer
```suggestion
 * Lager filter for fasetter med flere nivå, feks fylke og kommune. Kombinerer AND og OR.
```